packages/wdio-allure-reporter/src/utils.ts

```diff
@@ -19,18 +19,13 @@ export const getTestStatus = (test: TestStats | HookStats, config?: Options.Test
     }
 
     if (test.error) {
-        if (test.error.name && test.error.message) {
-            const message = test.error.message.trim()
-            return (test.error.name === 'AssertionError' || message.includes('Expect'))  ? 'failed' : 'broken'
+        if (test.error.message) {
+            const message = test.error.message.trim();
```

Comment:
@lacell75 could we simplify like this:
```
const message = test.error.message.trim().toLowerCase();
and then
message.startsWith('assertionerror') || message.includes('expect')
```
the same for `test.error.stack`

what do you think @lacell75 ?

Reply:
I agree . done